set.mm

```diff
@@ -32717,6 +32724,16 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
       ( wrex rspcedv mpd ) ACBDFJIABCDEFGHKL $.
   $}
 
+  ${
+    $d x ph $.  $d x B $.  $d x A $.
+    rr-rspce.1 $e |- ( ( ph /\ x = A ) -> ps ) $.
+    rr-rspce.2 $e |- ( ph -> A e. B ) $.
+    $( Prove a restricted existential.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    rr-rspce $p |- ( ph -> E. x e. B ps ) $=
```

Comment:
the labels starting with rr must be revised. the prefix rr are the initials of Rohan Ridenour, and were OK in his mathbox. In main, however, we do not have such prefixed labels.

Reply:
I removed the `rr-` prefix in the latest commit, and moved `rr-phpd` back into RR's mathbox. I hope labels I chose for some of the `rr-` theorems are okay.

Reply:
I am not happy with this label: this is not the deduction form of ~rspce, and it is not in deduction style at all (although it looks like). Looking at ~rspcimedv, this seems to be the deduction form of the theorem above, so it may be called ~rspcime.

Reply:
Named it ` rspcime `.